analyzers/src/SonarAnalyzer.ShimLayer.CodeGeneration/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
```

Comment:
Should we add the shared assembly to the project too?

Reply:
This is not needed since this assembly will not be shipped. It is used only during the project compilation.